woocommerce/woocommerce-android
# Added logic to schedule sale indefinitely

Fixes #2098 by adding the logic to schedule a sale without an end date.

#### Screenshots
<img width="300" src="https://user-images.githubusercontent.com/22608780/77341839-55f52b80-6d55-11ea-9551-31860ceb73ac.gif"/>

#### To test
- Click on a product from the Products TAB.
- Click on the Pricing section of the product (click on a product without a sale scheduled).
- Toggle the Schedule Sale option.
- Notice that the start date is set to the current date and the end date is empty.
- Click on the DONE button and click on the UPDATE button in product detail screen and verify that the sale is updated successfully.

Note that this PR only handles changes to schedule a sale indefinitely. The option to remove an end date from a sale will take place in another PR.

Update release notes:
